pkg/gds/client/profiles_test.go

```diff
@@ -0,0 +1,196 @@
+package client_test
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"testing"
+
+	"github.com/shibukawa/configdir"
+	"github.com/stretchr/testify/require"
+	"github.com/trisacrypto/directory/pkg/gds/client"
+	"gopkg.in/yaml.v2"
+)
+
+const profileYAML = "profiles.yaml"
+
+// Test that DefaultProfiles returns a Profiles object with some valid profiles.
+func TestDefaultProfiles(t *testing.T) {
+	profiles := client.DefaultProfiles()
+	require.NotNil(t, profiles)
+	require.True(t, strings.HasPrefix(profiles.Version, "v"))
+	require.NotEmpty(t, profiles.Profiles)
+	require.Contains(t, profiles.Profiles, profiles.Active)
+}
+
+// Test that GetProfilesFolder finds the correct folder.
+func TestGetProfilesFolder(t *testing.T) {
+	// Should default to the local path if it contains a profiles config
```

Comment:
So if I have a config file in my user dir and I run the tests locally it shouldn't find my profiles, it will just use the test profiles?